openlibrary/tests/core/test_processors_invalidation.py

```diff
@@ -33,7 +33,7 @@ def test_hook(self, monkeypatch):
             "key": "/templates/site.tmpl",
             "type": "/type/template"
         }
-        web.ctx.site.save(doc, timestamp=datetime.datetime(2010, 0o1, 0o1))
+        web.ctx.site.save(doc, timestamp=datetime.datetime(2010, 1, 1))
```

Comment:
Another change not related to Python 3

Reply:
$ __python3 -c "01"__  # --> SyntaxError: invalid token